src/Idephix/Extension/Slack/Slack.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Idephix\Extension\Slack;
+
+use Idephix\IdephixInterface;
+use Idephix\Extension\IdephixAwareInterface;
+
+/**
+ * Description of Slack wrapper
+ *
+ * @author dymissy
+ */
+class Slack implements IdephixAwareInterface
+{
+    /**
+     * @var \Idephix\IdephixInterface
+     */
+    private $idx;
+
+    private $settings;
+
+    public function __construct($args = array())
```

Comment:
in other extensions (see https://github.com/ideatosrl/Idephix/blob/master/src/Idephix/Extension/Project/Project.php) parameters are passed via retrieved via $idx. I think we should maintain consistency